src/wovnio/wovnphp/Logger.php

```diff
@@ -0,0 +1,97 @@
+<?php
+namespace Wovnio\Wovnphp;
+
+/**
+ * Logger implementation inspired by PSR3: https://www.php-fig.org/psr/psr-3/.
+ * Unlike PSR3, the `log` function is private and there are no constants for
+ * representing log levels.
+ */
+class Logger
+{
+  private static $logger = null;
+
+  public $prefix;
+  public $quiet;
+
+  public static function get()
+  {
+    if (self::$logger === null) {
+      self::$logger = new Logger();
+    }
+
+    return self::$logger;
+  }
+
+  public static function set($logger)
+  {
+    self::$logger = $logger;
```

Comment:
should we throw error if `$logger` is not instance of Logger?

Reply:
Yes we can. Currently it allows for duck typing, if our user want’s to use another PSR3 compatible logger for WOVN.php logs.

I don’t think adding an exception is a must have here and for the reason I just gave I prefer not.

Please feel free to argue if you disagree.

Reply:
Ah, thanks for the explanation. Let's keep the code then